src/Renderer/LayeredMaterial.js

```diff
@@ -420,4 +419,20 @@ LayeredMaterial.prototype.getLayerLevel = function getLayerLevel(layerType, laye
     }
 };
 
+LayeredMaterial.prototype.getLayerTextures = function getLayerTextures(layerType, layer) {
+    if (layerType === l_ELEVATION) {
+        return this.textures[l_ELEVATION];
+    }
+
+    const index = this.indexOfColorLayer(layer);
+
+    if (index !== -1) {
+        const count = this.getTextureCountByLayerIndex(index);
+        const textureIndex = this.getTextureOffsetByLayerIndex(index);
+        return this.textures[l_COLOR].slice(textureIndex, textureIndex + count);
+    } else {
+        return [];
```

Comment:
`throw new Error(`Requesting textures for invalid layer : ${layer});`
Also replace function param name : `layer` -> `layerId`